### PR TITLE
RELEASE: 0.2.1-DEMO-MAINNET

### PR DESCRIPTION
**Pull request by W3stside**
# DO NOT SQUASH

## Release v0.2.0-DEMO 
Release for the demo

Changelog:
* updated contracts to alpha-10
* re-enable mainnet
* fix url mainnet api
* update rinkeby block

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr83--gpv2swap.review.gnosisdev.com)**: Testing web app

**Review by Velenir — APPROVED** [review]

**Comment by W3stside** [comment]
Not sure what's up with the tests here.... @Velenir @anxolin

**Review by anxolin — APPROVED** [review]

File: cypress/integration/token-warning.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
+// TODO: this token can be added at any time, failing tests...
+const UNLISTED_TOKEN = '0x6c6ee5e31d828de241282b9606c8e98ea48526e2'
```
[the remark this answers is not in the document]
Comment (W3stside) [reply cypress/integration/token-warning.ts:2]: honestly have no idea, I think it's a token they added that we no longer support and it broke async somehow

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "@popperjs/core": "^2.4.4",
     "@reach/dialog": "^0.10.3",
     "@reach/portal": "^0.10.3",
-    "@reduxjs/toolkit": "^1.3.5",
+    "@reduxjs/toolkit": "^1.5.0",
```
[the remark this answers is not in the document]
Comment (W3stside) [reply package.json:13]: yes, Dima needed this to add the redux action checks inside his middleware. tho not strictly required it makes things easier there

**Comment by anxolin** [comment]
I'm not too convinced about updating redux versions like that. Anyways, are we merging this? to master, and then develop?

**Pull request merged**